user-registration-app/src/app/views/live-view/live-view.component.ts

```diff
@@ -44,10 +44,6 @@ export class LiveViewComponent implements OnInit {
   }
 
   ngOnInit() {
-    particlesJS.load('particles-js', 'assets/particles.json', function() {
```

Comment:
No more particles?

Reply:
I accidentally deleted it lmao, I'm gonna put it back